how-to/how-to-use-virtio-mem-with-kata.md

```diff
@@ -0,0 +1,49 @@
+# Kata Containers with `virtio-mem`
+
+- [Introduction](#introduction)
+- [Requisites](#requisites)
+- [Run a Kata Container with utilizing `virtio-mem`](#run-a-kata-container-utilizing-virtio-mem)
+
+## Introduction
+
+The basic idea of `virtio-mem` is to provide a flexible, cross-architecture memory hot plug and hot unplug solution that avoids many limitations imposed by existing technologies, architectures, and interfaces. More details can be found in https://lkml.org/lkml/2019/12/12/681.
+
+Kata Containers with `virtio-mem` support memory resize.
+
+## Requisites
+
+Kata Containers with `virtio-mem` requires Linux and the QEMU that support `virtio-mem`.
```

Comment:
Can you make the following clear:

- Users require a particular Linux version running on the host.
- The versions numbers of Linux and QEMU required ("modified Linux x.y.z" or similar)
- Why the QEMU fork and Linux forks are required.
- Add a warning that this are not the official versions of QEMU and Linux, "your mileage may vary", etc, etc.
- Any details on possible timing for when the features in the forks may make it into the mainlines?

Reply:
done.